<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntryBox</class>
 <widget class="QDialog" name="DataEntryBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Website</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="websiteDATA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameDATA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordDATA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
